### DISCUSSION TITLE
Mi Band WatchFace Editor GUI

### DISCUSSION THREAD

**Discussion by Johnson070** [Johnson070/MiBand-5-watchface-editor#13] [General]
1.0.1.3 (2021-04-24)

- Bug fixes

- Fix number form 

- Fix add preview mi band 6

- Add full support MiBand 6(With building a bin file yet)

- Add simple select images(Hold down the left mouse button to select, and to delete with shift)

- Animation preview(use carefully, the program uses more than 100 MB of RAM)

Thank you for downloading the program.

If you find a bug, please write to github or telegram: @vveber

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnson070/MiBand-5-watchface-editor/releases/tag/1.0.1.3'>Mi Band WatchFace Editor GUI</a>.</em>

**Comment by Jaszuro** [comment]
Hi & thanks.
However Avira detects a trojan in WatchFace6\WatchFace.exe. Probably a false positive, but it does not detect it in the non-6 file, so i'm wondering... Submitted the file to Avira, waiting for their reply.

**Comment by Johnson070** [comment]
#16 check virustotal
